deploy/kubernetes/metricbeat/README.md

```diff
@@ -161,53 +141,41 @@ $ kubectl get pods -o wide --namespace=kube-system | grep ^metricbeat
 
 To visualize your data in Kibana, we will use Elastic Cloud. For that do the following:
 
-<ol>
-<li>
 
-[Log in](https://cloud.elastic.co/home) to your Elastic Cloud account.
-</li>
-<li>
 
-Create a [deployment](https://www.elastic.co/guide/en/cloud/current/ec-create-deployment.html). Make sure to save the credentials.
-</li>
-<li>
+1. [Log in](https://cloud.elastic.co/home) to your Elastic Cloud account.
 
-On the deployment overview page, copy down the Cloud ID.
-</li>
-<li>
+2. Create a [deployment](https://www.elastic.co/guide/en/cloud/current/ec-create-deployment.html). Make sure to save the credentials.
 
-Set the environment variables on the [metricbeat-daemonset.yaml](metricbeat-daemonset.yaml) file:
+3. On the deployment overview page, copy down the Cloud ID.
```

Comment:
since you mentioned to copy the cloud Id, maybe we should mention as well where to get the clout Auth variables from (eg. during the deployment process, it provides you those variables or it allows you to download a file locally.